.github/workflows/go.yaml

```diff
@@ -7,8 +7,12 @@ on:
       - autoupdate/strict
       - 'release-[0-9]+.[0-9]+'
       - 'release-[0-9]+.[0-9]+-strict'
+    paths:
+      - '**/*.go'
```

Comment:
careful, this might miss things. we also need paths like .github workflows (e.g. in case the workflow changes)